### DISCUSSION TITLE
Non label on delete modal

### DISCUSSION THREAD

**Discussion by eaziweb95** [filamentphp/filament#2701] [Help] [answered]
### Package

filament/tables

### Package Version

2

### Laravel Version

9

### Livewire Version

9

### PHP Version

8

### Bug description

The delete modal displays 
filament::resources/pages/list-records.table.actions.delete.modal.heading
Êtes-vous sûr de vouloir faire cela ?

**Comment by eaziweb95** [comment]
Thanks to resolve this issue

**Reply by danharrin** [reply]
I don't know what the issue is

**Reply by eaziweb95** [reply]
No translation on filament::resources/pages/list-records.table.actions.delete.modal.heading

**Comment by eaziweb95** [comment, marked as answer]
![delete](https://user-images.githubusercontent.com/58070424/174097395-a0fde13e-472c-4518-abfe-63cfc879dba1.jpg)

**Reply by danharrin** [reply]
I think you need to run `php artisan filament:upgrade` to clear your views

**Reply by eaziweb95** [reply]
There is a lack of translation. I was obliged to add this code in the list-records.php under vendor/filament/filament/resources/lang/fr/resources/pages

                'modal' => [
                    'heading' => 'Supprimer :label',
                ],

Same issue on Wizard.  I was obliged to add this code the components.php under vendor/filament/forms/resources/lang/fr

    'wizard' => [
        'buttons' => [
            'next_step' => [
                'label' => 'Suivant',
            ],
            'previous_step' => [
                'label' => 'Précédent',
            ]
        ]
    ]

**Reply by danharrin** [reply]
I checked the codebase and that translation string is not used anymore, which suggests your views are out of date. Did you maybe publish the views to /resources/views/vendor?
